### PR TITLE
0.7 updates

### PR DESCRIPTION
**Pull request by quinnj**

### CODE REVIEW

**Comment by omus** [comment]
Why not just wait for `Compat.Unicode`?

**Comment by quinnj** [comment]
Because these are super trivial uses and this package doesn't rely on Compat currently anyway.

File: src/DecFP.jl [hunk 1]
```diff
@@ -3,7 +3,7 @@ export Dec32, Dec64, Dec128, @d_str, @d32_str, @d64_str, @d128_str
 
 const libbid = joinpath(dirname(@__FILE__), "..", "deps", "libbid$(Sys.WORD_SIZE)")
 
-const _buffer = Vector{UInt8}(1024)
+const _buffer = fill(0x00, 1024)
```
Comment (stevengj) [inline src/DecFP.jl]: Wait, we can no longer allocate uninitialized arrays in 0.7?

Comment (quinnj) [reply src/DecFP.jl]: we can, it would be like `Vector{UInt8}(uninitialized, 1024)`, I just prefer the terseness of fill now.

Comment (stevengj) [reply src/DecFP.jl]: 🤢

**Pull request merged**